README.md

```diff
@@ -183,7 +183,7 @@ The results will be output in `./benchmarks/results.md`.
 
 ## strictObjectIDCoercion flag
 
-In version 1.17.0, where the id is of string type, is being converted to ObjectID when the string length is 12 or 24 and has the format of an ObjectID i.e /^[0-9a-fA-F]{24}$/. To avoid this issue, the strictObjectIDCoercion flag should be set to true in the model-definition file.
+In version 1.17.0, where the id of string type is being converted to ObjectID, when the string length is 12 or 24 and has the format of an ObjectID i.e /^[0-9a-fA-F]{24}$/. To avoid this issue, the strictObjectIDCoercion flag should be set to true in the model-definition file.
```

Comment:
Suggestion: `In version 1.17.0, If you have an `id` property of type string, it will be converted to ObjectID when its length is 12 or 24 and has the format of an ObjectID i.e /^[0-9a-fA-F]{24}$/.`